lib/bluetooth-serial-port.js

```diff
@@ -151,7 +154,11 @@
     util.inherits(BluetoothSerialPortServer, EventEmitter);
     exports.BluetoothSerialPortServer = BluetoothSerialPortServer;
 
-    BluetoothSerialPortServer.prototype.listen = function (uuid, listeningChannel, successCallback, errorCallback) {
+    BluetoothSerialPortServer.prototype.listen = function (successCallback, errorCallback, options) {
+        options = options || {};
```

Comment:
Here you should test if `errorCallback` is a function or not because both `errorCallback` and `options` are optional parameters. If the function is called like `listen(function() { /* my callback */}, { uuid: '1101' })` the code below will not work and will result in an error in case `errorCallback` is called.